packages/neos-ui/src/Sagas/UI/ContentCanvas/index.js

```diff
@@ -15,7 +16,16 @@ function * watchNodeCreated() {
         yield put(actions.UI.ContentCanvas.setSrc($get('uri', node)));
     });
 }
+/**
+ * Load newly created page into canvas
+ */
+function * watchCanvasUpdateToChangeTitle() {
+    yield * takeLatest(actionTypes.UI.ContentCanvas.STOP_LOADING, () => {
+        document.title = iframeDocument().title;
```

Comment:
Just wondering, maybe it would make sense to prefix backend with something, not to confuse it with frontend website pages?